tests/Console/Output/Progress/DotsOutputTest.php

```diff
@@ -179,7 +179,7 @@ public static function provideProcessProgressOutputCases(): array
     public function testSleep(): void
     {
         $this->expectException(\BadMethodCallException::class);
-        $this->expectExceptionMessage('Cannot serialize PhpCsFixer\Console\Output\DotsOutput');
+        $this->expectExceptionMessage('Cannot serialize PhpCsFixer\Console\Output\Progress\DotsOutput');
```

Comment:
use `DotsOutput::class`

Reply:
FYI This is code from original `ProcessOutput`, not mine, so I just adjusted it to new FQCN but did not change the approach. But ok, I can change it.

Reply:
got you, happy with both then - agree it could be out of scope of PR. hard to notice when github shows as new file 😓 
[ultimately I believe we should use `::class`, like we already doing in majority of places, but it wasn't easy to find all cases to improve]